languageServer/index.d.ts

```diff
@@ -0,0 +1,4 @@
+import { DocumentFilter } from 'vscode-languageserver'
```

Comment:
Why not just let `tsc` emit the declarations for us? This file is pretty simple, so I don't think it's really necessary, but would it help making it future proof?